examples/binary-operators/binary_operators.bal

```diff
@@ -0,0 +1,65 @@
+import ballerina/io;
+
+public function main() {
+    int a = 10;
+    int b = 20;
+
+    // Adds two integers.
```

Comment:
```suggestion
    // Add two integers.
```

Change everywhere.

Also better to assign to a variable and then print to demonstrate static type.